pinot-core/src/main/java/org/apache/pinot/core/transport/grpc/GrpcQueryServer.java

```diff
@@ -88,6 +94,20 @@ public void submit(ServerRequest request, StreamObserver<ServerResponse> respons
       return;
     }
 
+    // Table level access control
+    GrpcRequesterIdentity requestIdentity = new GrpcRequesterIdentity(request.getMetadataMap());
+    if (!_accessControl.hasDataAccess(requestIdentity, queryRequest.getTableNameWithType())) {
+      Exception unsupportedOperationException = new UnsupportedOperationException(
+          String.format("No access to table %s while processing request %s: %s from broker: %d",
+              queryRequest.getTableNameWithType(), queryRequest.getRequestId(),
+              queryRequest.getQueryContext(), queryRequest.getBrokerId()));
+      LOGGER.error("Table not found: {}", queryRequest.getTableNameWithType(), unsupportedOperationException);
```

Comment:
This will not print the stack trace, it resolves to 
```java
    /**
     * Log a message at the ERROR level according to the specified format
     * and arguments.
     * <p/>
     * <p>This form avoids superfluous object creation when the logger
     * is disabled for the ERROR level. </p>
     *
     * @param format the format string
     * @param arg1   the first argument
     * @param arg2   the second argument
     */
    public void error(String format, Object arg1, Object arg2);
```

instead of
```java
    /**
     * Log an exception (throwable) at the ERROR level with an
     * accompanying message.
     *
     * @param msg the message accompanying the exception
     * @param t   the exception (throwable) to log
     */
    public void error(String msg, Throwable t);
```

I recently had to change one of these to identify the cause of an error: #7948

Reply:
make sense. Will fix this

Reply:
Discussed with @richardstartin offline, and both methods are okay and should have the same behavior